app/Jobs/PollForMediaWikiJobsJob.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use Illuminate\Support\Facades\Http;
+
+class PollForMediaWikiJobsJob extends Job
+{
+    public function handle(): void
+    {
+        $wikis = Wiki::all()->pluck('domain');
+        foreach ( $wikis as $wikiDomain ) {
+            if ($this->hasPendingJobs( $wikiDomain )) {
+                $this->enqueueWiki( $wikiDomain );
+            }
+        }
+    }
+
+    private function hasPendingJobs( string $wikiDomain ): bool
```

Comment:
I wonder if there is a way to do this with a little less HTTP involved? Or is this the go-to solution for us?

Reply:
there is a way: we have sufficient credentials in this place to read the db. I personally prefer this though since in my opinion the http is the more stable interface